DPGAnalysis/SiStripTools/python/apvcyclephaseproducerfroml1ts2010_cfi.py

```diff
@@ -9,5 +9,7 @@
                                                                ),
                            defaultPhases = cms.vint32(30,30,30,30),
                            l1TSCollection = cms.InputTag("scalersRawToDigi"),
+                           tcdsRecordLabel= cms.InputTag("unpackTcds","tcdsRecord"),
+                           forceSCAL = cms.bool(False),
```

Comment:
at @vischia quoting Matti here:

> In general the default values should correspond to Run 1, with customizations for later periods done with era modifiers. Parameters should be tracked if they can have any impact on results/physics.

Isn't this breaking the rule?

Reply:
From what I understand in Run1 TCDS was not available, so SCAL will be used anyways, regardless of the value of forceSCAL.

The purpose of forceSCAL is to force usage of SCAL when TCDS is available, not of pointing out whether TCDS was available or not.

Reply:
I think what Andrea had in mind is something different. We must ensure that Run1 and Run2 do use SCAL no matter what.

Reply:
OK, so I put it to True by default, and then at RunIII one can activate it, would that be OK?

Reply:
sounds good to me. Thanks!

Reply:
Or I can put it to something that gives True for Run1 and Run2, and false for Run3, if there is anything with this behaviour. How do I access Run information at the level of the config files?

Reply:
> sounds good to me. Thanks!

Oh OK, then I'll do that, thanks!

Reply:
I think the default should be to true, and using the Run3 era modifier it should be allowed to take TCDS when available.

Reply:
Note that SCAL will no longer exist in run 3. Thus, you have to use TCDS in the future.

Reply:
@mommsen that's why in the run3 era that parameter is set to False:
https://github.com/cms-sw/cmssw/blob/9cb7235ee1351aed1f9a8f2a15c9b6d435e495f0/DPGAnalysis/SiStripTools/python/apvcyclephaseproducerfroml1ts_cfi.py#L16-L17